content/master/cli/command-reference.md

```diff
@@ -380,11 +377,7 @@ in the package documentation.
 
 ### xpkg login
 
-Use `xpkg login` to authenticate to `xpkg.upbound.io`, the 
-[Upbound Marketplace](https://marketplace.upbound.io/) container registry.
-
-[Register with the Upbound Marketplace](https://accounts.upbound.io/register) 
-to push packages and create private repositories. 
+Use `xpkg login` to authenticate to `xpkg.crossplane.io`, the [crossplane-contrib](https://github.com/orgs/crossplane-contrib/packages) GitHub Container Registry used by the Crossplane project.
```

Comment:
this `xpkg login` command hasn't been updated, and it's not entirely clear what/how to update it yet either. I can imagine we add an extra arg that specifies which registry to login to, so all vendors have this extension point available - but that's not being done super short term.

I would keep the change to remove the link to register, but maybe we can make it more generic for the login part. e.g.,:
```
Use `xpkg login` to authenticate to registries that host Crossplane packages.
```
```